README.md

```diff
@@ -122,35 +134,39 @@ cls_explainer.visualize("distilbert_negative_attr.html")
 
 <img src="images/distilbert_example_negative.png" width="80%" height="80%" align="center" />
 
-Getting attributions for different classes is particularly insightful for multiclass problems as it allows you to inspect model predictions for a number of different classes and sanity check that the model is "looking" at the right things.
+Getting attributions for different classes is particularly insightful for multiclass problems as it allows you to inspect model predictions for a number of different classes and sanity-check that the model is "looking" at the right things.
 
-For a detailed example of this please checkout this [multiclass classification notebook](notebooks/multiclass_classification_example.ipynb)
+For a detailed explanatioon of this example please checkout this [multiclass classification notebook.](notebooks/multiclass_classification_example.ipynb)
 
 ## Future Development
 
-This package is still in its early days and there is hopefully much more planned. For a 1.0.0 release I'm aiming to have:
+This package is still in its early days and there is much more planned. For a 1.0.0 release we're aiming to have:
 
-- Proper documentation site
+- Clean and thorough documentation
 - Support for Question Answering models
 - Support for NER models
-- Support for Multiple Choice models (not sure how feasible this is)
-- Ability to show attributions for each layer rather than a summary of all layers
+- Support for Multiple Choice models
+- Ability to show attributions for each layer, rather than a summary of all layers
 - Additional attribution methods
 - In depth examples
-- Get a nice logo
-- and more...
+- A nice logo
+- and more... feel free to submit your suggestions!
 
 ## Questions / Get In Touch
 
 The main contributor to this repository is [@cdpierse](https://github.com/cdpierse).
 
-If you have any questions, suggestions, or would like to make a contribution (please do 😁) please get in touch at charlespierse@gmail.com
+If you have any questions, suggestions, or would like to make a contribution (please do 😁), feel free to get in touch at charlespierse@gmail.com
+
+I'd also highly suggest checking out [Captum](https://captum.ai/) if you find model explainability and interpretability interesting. They are doing amazing and important work. In fact, this package stands on the shoulders of the the incredible work being done by the teams at [Pytorch Captum](https://captum.ai/) and [Hugging Face](https://huggingface.co/) and would not exist if not for the amazing job they are both doing in the fields of NLP and model interpretability respectively.
```

Comment:
I moved the credit to Captum and Hugging Face here - it flows much more nicely and you're still giving the credit where its due.

Reply:
Much better positioning. Thanks for all of these comments.